controllers/newsControllers.js

```diff
@@ -0,0 +1,13 @@
+const { fetchTopics } = require("../models/newsModels");
+
+function fetchAllTopics(request, response) {
+  fetchTopics()
+    .then((data) => {
+      console.log(data);
```

Comment:
again here console logs, I think calling it data is fine, but could be a little more descriptive, we know that its not just any data, it should be some topics :)